andela/rogue-backend
# #167727617 Setup Unit Testing

### What does this PR do?
- Sets up Test runner for the project application

### Description of Task Taken to Complete?
- Installs mocha, chai and chai-http
- Creates test folder to house all test files


### How should this be manually Tested?
- `git clone` this repo
- `git checkout` ch-setup-unit-testing-167727617
- Run `npm test` to see test pass


### Any Background Context you want to provide?
- The Test folder created will house all test cases for every feature implemented


### What are the relevant Pivotal Tracker Stories?
- [167727617](https://www.pivotaltracker.com/story/show/167727617)
